integration_test/tests/power_bi_tests.rs

```diff
@@ -531,4 +531,185 @@ mod integration {
             );
         }
     }
+
+    /// Test PowerBI flow for listing tables
+    ///     - SQLAllocHandle(SQL_HANDLE_STMT)
+    //      - SQLTablesW(null pointer, null pointer, null pointer, "TABLE,VIEW")
+    //      - SQLGetFunctions(SQL_API_SQLFETCHSCROLL)
```

Comment:
The flow from SQLGetFunctions to SQLFetch (exclusive) is the same than for test_data_retrieval()
What about we extract the duplicate part in a helper?
It also true for the SQLFetch to SQLMoreResults part. I can see it be bit more cumbersome since it needs to deal with the target C type for each column, but passing in an array of the targeted C types for each column could be enough.

Reply:
That's a good suggestion, I added helper functions for these two sections.